resources/zib/zib-HeartRate.HeartbeatRegularity.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-HeartRate.HeartbeatRegularity" />
+  <text>
+    <status value="empty" />
+    <div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div>
+  </text>
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-HeartRate.HeartbeatRegularity" />
+  <name value="ZibHeartRateHeartbeatRegularity" />
+  <title value="zib HeartRate.HeartbeatRegularity" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Regularity of the heartbeat." />
+  <purpose value="This Observation resource represents the HeartbeatRegularity concept (NL-CM:12.3.5) of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) HeartRate v3.4 (2020EN)](https://zibs.nl/wiki/HeartRate-v3.4(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-heartrate-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/HeartRate-v3.4(2020EN)" />
+    <name value="zib HeartRate-v3.4(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <comment value="This Observation resource represents the HeartbeatRegularity concept (NL-CM:12.3.5) of zib HeartRate. It can be used together with the [zib-HeartRate](http://nictiz.nl/fhir/StructureDefinition/zib-HeartRate) profile on the Observation resource, which can be referenced using `Observation.hasMember`." />

```

Comment:
Shouldn't the latter part be 'wherein it can be referenced using `Observation.hasMember`', since the reference goes from zib-HeartRate to this profile?

Reply:
The latter part of the comment is currently not yet part of https://informatiestandaarden.nictiz.nl/wiki/FHIR:Vissue-hasMember_FHIR_Profiling_Guidelines_R4, but I haven't added it yet due to the sentence possibly being changed based on the above.

Reply:
I would like to propose: "..., which groups references to the current resource on `Observation.hasMember`" instead, as it draws the attention to the grouping mechanism. What do you think?

Reply:
I agree with drawing attention to the grouping mechanism. But wouldn't the above imply that you could have multiple references to the same resource? Of course FHIR doesn't prohibit this, but it is not in line with the cardinality in the zib. Should it then be something like '..., which groups references to the current and other resources on `Observation.hasMember`'?

Actually the above raises another question: in the previous modeling we respected the maximum cardinalities of the zib, but in the `.hasMember` approach we do not enforce e.g. the HeartbeatRegularity to be present at most once. Is that something we need to consider by for instance adding slices for each member?

Reply:
I think you're right on both accounts.
My proposal would be to make a profile slice and add the following comment on the `.definition` of each slice: "The [concept name] concept ([concept id]) of [model name], represented as a distinct Observation resource." The comment on `.hasMember` itself is not needed anymore in this case.

Reply:
Sounds perfect!